### ISSUE TITLE
Suggest: indexing PSR

### ISSUE THREAD

**Issue by Mahmoudz** [php-fig/php-fig.github.com#227]
* PSR 0 Autoloading Standard
* PSR 4 Autoloading Standard
___
* PSR 1 Basic Coding Standard
* PSR 2  Coding Style Guide
* PSR 12 Extended Coding Style Guide
* PSR 5 PHPDoc Standard
___
* PSR 6 Caching Interface
* PSR 16 Simple Cache
___
* PSR 7 HTTP Message Interface
* PSR 15 HTTP Middlewares
* PSR 17 HTTP Factories
___
* PSR 3 Logger Interface
* PSR 11 Container Interface
* PSR 8 Huggable Interface
___
...

**Comment by railto** [comment]
I agree that this would be nice, making sure of the following 2 things:

1, Grouped PSRs should be in numeric order
2, Have a 'Group Title' at the top of each grouping to make it clear what the group is for

**Comment by stefanotorresi** [comment]
Good idea; it needs some Sculpin-fu, though, so that we can drop the inclusion of the raw `index.md` file from `fig-standards` repo and use a dynamic table view.
Anyone feel free to pick this up, atm I don't have much time.
